brave/src/main/java/brave/propagation/SamplingFlags.java

```diff
@@ -110,13 +110,25 @@ public final boolean debug() {
   }
 
   @Override public String toString() {
-    return "SamplingFlags(sampled="
-      + sampled()
-      + ", sampledLocal="
-      + sampledLocal()
-      + ", debug="
-      + debug()
-      + ")";
+    return toString(flags);
+  }
+
+  static String toString(int flags) {
+    StringBuilder result = new StringBuilder();
+    if ((flags & FLAG_DEBUG) == FLAG_DEBUG) {
+      result.append("DEBUG");
+    } else  if ((flags & FLAG_SAMPLED_SET) == FLAG_SAMPLED_SET) {
+      if ((flags & FLAG_SAMPLED) == FLAG_SAMPLED) {
+        result.append("SAMPLED");
+      } else {
+        result.append("NOT_SAMPLED");
+      }
+    }
+    if ((flags & FLAG_SAMPLED_LOCAL) == FLAG_SAMPLED_LOCAL) {
+      if (result.length() > 0) result.append('|');
+      result.append("SAMPLED_LOCAL");
```

Comment:
I think this can produce `NOT_SAMPLED|SAMPLED_LOCAL` which seems a bit weird.

Reply:
should we rename to NOT_SAMPLED_REMOTE?